apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ProjectMetricsResourceTest.java

```diff
@@ -908,14 +926,15 @@ private Pair<List<String>, List<List<FeedbackScoreBatchItemThread>>> createThrea
     @DisplayName("Token usage")
     @TestInstance(TestInstance.Lifecycle.PER_CLASS)
     class TokenUsageTest {
+
         @ParameterizedTest
         @EnumSource(TimeInterval.class)
         void happyPath(TimeInterval interval) {
             // setup
             mockTargetWorkspace();
 
             Instant marker = getIntervalStart(interval);
-            String projectName = RandomStringUtils.randomAlphabetic(10);
+            String projectName = RandomStringUtils.secure().nextAlphabetic(10);
```

Comment:
secure().nextAlphabetic is better 🔥 

We use with hardcoded value 10, everywhere

Perhaps, we should export to an utility function?

To generate a randomName (projectOrWorkspace)